### ISSUE TITLE
Is this suppose to be a public repo :)

### ISSUE THREAD

**Issue by bogomil** [portworx/action-build-changelog#2]
Is this suppose to be a public repo :)

**Comment by fmilichovsky** [comment]
Yeah, for reusability as a github action, and it doesn't have any sensitive business logic.
But we need to add some fancy license stuff etc., thanks for pointing it out :)
